cuplaConfig.cmake

```
@@ -22,7 +22,7 @@
 # Required cmake version.
 ################################################################################
 
-cmake_minimum_required(VERSION 3.11.0)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.11.4)
```

lowercase is sexy, modern, and consistent with the other CMake code below:
```suggestion
cmake_minimum_required(VERSION 3.11.4)
```

---

copy past copy past copy past^^